x-pack/plugins/security_solution/public/common/containers/source/index.tsx

```diff
@@ -126,7 +126,7 @@ export const useFetchIndex = (
   const { data, notifications } = useKibana().services;
   const abortCtrl = useRef(new AbortController());
   const previousIndexesName = useRef<string[]>([]);
-  const [isLoading, setLoading] = useState(true);
+  const [isLoading, setLoading] = useState(false);
```

Comment:
Tested the few other touch points around this (add/edit exceptions, overview, etc) and all looks 👍.